src/reminders.py

```diff
@@ -125,15 +126,17 @@ def strip_conjunctions(words):
 
 def parse_remind_me(time_and_reminder):
     words = time_and_reminder.split(" ")
-    timereg_parts = [r"((?P<{}>\d*){})?".format(cha, cha) for cha in "ydhms"]
+    timereg_parts = [r"((?P<{}>\d*){})?".format(cha, cha) for cha in "yMdhms"]
     timereg = re.compile(r"^"+"".join(timereg_parts)+r"$")
     short_match = re.search(timereg, words[0])
     if short_match:
-        years, days, hours, minutes, seconds = [int(val) if val else 0
+        years, months, days, hours, minutes, seconds = [int(val) if val else 0
                                                 for val
-                                                in short_match.group(*"ydhms")]
-        total_seconds = (years*31557600 + days*86400 + hours*3600 + minutes*60
-                         + seconds)
+                                                in short_match.group(*"yMdhms")]
+        relative_delta = relativedelta(years=years,months=months,days=days,
+                                hours=hours,minutes=minutes,seconds=seconds)
+        utcnow = datetime.utcnow()
+        total_seconds = (utcnow + relative_delta - utcnow).total_seconds()
```

Comment:
```suggestion
        total_seconds = relative_delta.total_seconds()
```

Since `utcnow + relative_delta - utcnow` is the same as `relative_delta`, right?

Reply:
This line is in here because a `relativedelta` object cannot be converted into seconds. By adding the delta to the current time and then subtracting it, we turn it into a timedelta.